tests/unit_tests/ir2verilog_test.py

```diff
@@ -119,3 +119,15 @@ def test_later_occurences_of_defined_names_are_scoped(self):
             template.substitute({"module_name": "FILTER_FIR_HALF"})
             == "`define FILTER_FIR_HALF_DATA_WIDTH 8\nFILTER_FIR_HALF_DATA_WIDTH + 10"
         )
+
+    def test_can_replace_arrays(self):  #TODO maybe get better name
+        template = (
+            TemplateDirector()
+            .set_prototype("localparam DATA = {'d2, 'd3};")
+            .localparam("DATA")
+            .build()
+        )
+        assert (
+            template.substitute({"DATA": "'d5"})
+            == "localparam DATA = 'd5;"
+        )
```

Comment:
Looks good 👍. Do you think we need another test for replacing an array by another array?